sw/airborne/state.h

```diff
@@ -580,6 +582,14 @@ static inline void stateSetPositionUtm_f(struct UtmCoor_f *utm_pos)
   state.pos_status = (1 << POS_UTM_F);
 }
 
+/// Set altitude above ground level (float).
+static inline void stateSetAltAgl_f(float agl)
+{
+  state.alt_agl_f = agl;
+  /* clear bits for all position representations and only set the new one */
```

Comment:
I don't think that it is a good idea to clear all other bits here as you don't want to invalidate the rest of the position...